docker-compose.prod.yml

```diff
@@ -52,17 +54,16 @@ services:
     restart: always
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-    command: --cleanup=true
+    command: --cleanup=true --interval 60
 
 networks:
   app-network:
     driver: bridge
 
-volumes:
-  app-data:
-
 secrets:
   cert_chain:
     file: ./certs/certificate.pem
   priv_key:
     file: ./private/private.key
+  twiliorecording_key:
+    file: ./private/twiliorecording.key
```

Comment:
@pauljohanneskraft Added this in preparation for your next PR; should already be on the server to be able to test things there. Would be great if we can add some documentation surrounding this.